@xen-orchestra/web/src/components/account-menu/AccountMenuTrigger.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <button
+    v-tooltip="{ content: $t('account-organization-more'), placement: 'bottom-end' }"
```

Comment:
I suggest to disable the tooltip when the menu is open.

```suggestion
    v-tooltip="active ? false : { content: $t('account-organization-more'), placement: 'bottom-end' }"
```

In addition, the tooltip text should be updated to remove "coming soon".